server/trillian_log_server/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/monitoring"
-	"github.com/google/trillian/quota"
+	mysqlq "github.com/google/trillian/quota/mysql"
```

Comment:
There could be advantages in having all the mysql code together. As we currently only have one implementation we can probably worry about this later.